scripts/run_kickstart_tests.sh

```diff
@@ -163,7 +178,7 @@ function find_tests() {
     local newtests=""
     local skipped_tests=""
     for f in ${tests}; do
-        if [[ "$SKIP_TESTTYPE" != "" && "$(grep TESTTYPE= ${f})" =~ "${SKIP_TESTTYPE}" ]]; then
+        if should_skip_test ${f}; then
             skipped_tests+="${f}"
```

Comment:
I was just wondering why the variable when it is not used / passed anywhere.

Reply:
Previously I planned to print the skipped tests as well as the selected tests but this turns out to be non trivial due to how bash functions work. It's pretty hard to return two lists from a single function and also functions apparently can't write to global variables. 

So let's just leave this as-is, we plan to replace this shell script with Python anyway.